### PR TITLE
Adds Loop to the Dir 

### PR DESCRIPTION
**Pull request by jvanderaa**
Adds loop to attempt to get the dir output 5 times before failing to detect OS

### CODE REVIEW

**Review by jmcgill298 — APPROVED** [review]

File: pyntc/devices/ios_device.py [hunk 1]
```diff
@@ -77,13 +77,21 @@ def _get_file_system(self):
...
-        raw_data = self.show("dir")
-        try:
-            file_system = re.match(r"\s*.*?(\S+:)", raw_data).group(1)
-        except AttributeError:
-            raise FileSystemNotFoundError(hostname=self.facts.get("hostname"), command="dir")
+        # Set variables to control while loop
+        counter = 0
+
+        # Attempt to gather file system
+        while counter < 5:
```
Comment (mundruid) [inline pyntc/devices/ios_device.py:84]: Why specifically check 5 times? Can you please variabilize this with a constant to be able to change if needed?

**Review by mundruid — CHANGES_REQUESTED** [review]
Hi Josh, the changes look good but the pipeline is failing black. Can you please fix the issues? Thanks!

File: pyntc/devices/ios_device.py (hunk 1 above)
Comment (jvanderaa) [reply pyntc/devices/ios_device.py]: Made into a variable/constant. 5 times as a number. Could maybe be 3. In the test it only worked 1 out of the 4 times. The other 3 had the error occur.

On the format front, not sure why the autoformat of VSCode would have different black settings than the project. I'm going to look into that yet.

**Comment by jmcgill298** [comment]
Thanks @jvanderaa 

Thanks @mundruid for keeping us honest :)

**Pull request merged**